src/stream_processor/CMakeLists.txt

```diff
@@ -1,5 +1,3 @@
-project(stream-processor)
```

Comment:
one way to enforce C only would be to add the language after the project name, e.g:

```
project(stream-processor C)
```

Reply:
I know, and my original patch was like this.
See https://github.com/fluent/fluent-bit/pull/7254#issuecomment-1520273947,  which made me to drop the project() line all together. I can restore that commit.

About the prefix, I used the prefixes I saw in git blame but ok, I can also change that.

Reply:
ahh sorry about that, I misread it